stable_baselines/a2c/a2c.py

```diff
@@ -239,7 +239,7 @@ def learn(self, total_timesteps, callback=None, seed=None, log_interval=100, tb_
                 obs, states, rewards, masks, actions, values, ep_infos, true_reward = runner.run()
                 ep_info_buf.extend(ep_infos)
                 _, value_loss, policy_entropy = self._train_step(obs, states, rewards, masks, actions, values,
-                                                                 self.num_timesteps // (self.n_batch + 1), writer)
+                                                                 update - 1, writer)
```

Comment:
If you don't use `self.num_timesteps`, then it will be wrong when you call `model.learn()` with `reset_num_timesteps=False`.

Reply:
Thanks, I didn't notice that. I've fixed it in new commits.